lib/embedded_ansible.rb

```diff
@@ -120,6 +122,11 @@ def self.appliance_disable
   private_class_method :appliance_disable
 
   def self.container_start
+    if VMDB::Util.http_proxy_uri(:embedded_ansible)
+      _log.error("Can't set proxy for Embedded Ansible Tower in container environment")
+    elsif VMDB::Util.http_proxy_uri
+      _log.warn("Global proxy settings will not work for Embedded Ansible Tower in container environment")
```

Comment:
I'm not sure that this is the case, but I don't think this would really be needed for the container case either way.

Reply:
For container case, we can't modify it's settings.py and set the proxy settings since the ansible tgwer already set up in container. You mean we don't need to consider this case?

Reply:
> we can't modify it's settings.py and set the proxy settings since the ansible tgwer already set up in container.

Of course, you're right here.

> You mean we don't need to consider this case?

I mean that in OpenShift at least, it's likely that we will have more permissive network that will allow us to access the resources we need. I'm just thinking that a proxy will be a less common use case in containers.